tests/unit/lms/services/jwt_test.py

```diff
@@ -57,24 +61,90 @@ def test_encode_with_secret(self, svc, jwt):
         )
         assert encoded_jwt == jwt.encode.return_value
 
+    def test_decode_lti_token_with_no_kid(self, svc):
+        jwt_str = self.encode_jwt({"key": "value"}, headers={"key": "value"})
```

Comment:
Slight whiplash here with some tests using the real `jwt` and others mocking it. I don't mind it, but I'd be interested to know why we did it one way here and another in other places.